NewsApi/app.py

```diff
@@ -1,10 +1,29 @@
 from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
 from flask_marshmallow import Marshmallow
+from flask_bcrypt import Bcrypt
+from flask_jwt_extended import JWTManager
+from flask_apiexceptions import JSONExceptionHandler
+from flask_login import LoginManager
+
 
 app = Flask(__name__)
-app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///database.sqlite'
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///db.sqlite'
+app.config["JWT_SECRET_KEY"] = "1234567890"  # Change this!
+app.config["SECRET_KEY"] = "1234567890"  # Change this!
```

Comment:
We discussed in class how to deal with secrets and configuration variables. Maybe you can apply that here.